php-pre-challenge1/index.php

```diff
@@ -0,0 +1,13 @@
+<?php
+for ($i = 1; $i <= 100; $i++) {
+  if ($i % 3 == 0 && $i % 5 == 0) {
+    echo '3の倍数であり、5の倍数<br>';
+  } elseif ($i % 3  == 0) {
```

Comment:
```suggestion
  } elseif ($i % 3  === 0) {
```
厳密比較 '===' にしてください。
どちらでも動く場合が多いですが、昨今は '===' だけを使うようにするのがおすすめです。